client/signup/steps/site-title/index.jsx

```diff
@@ -0,0 +1,201 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import isEmpty from 'lodash/isEmpty';
+import { getSiteTitle } from 'state/signup/steps/site-title/selectors';
+import { setSiteTitle } from 'state/signup/steps/site-title/actions';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import LoggedOutForm from 'components/logged-out-form';
+
+import formState from 'lib/form-state';
+import ValidationFieldset from 'signup/validation-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormButton from 'components/forms/form-button';
+import FormTextInput from 'components/forms/form-text-input';
+
+import ExampleSiteTitle from './example-site-title';
+
+const SiteTitle = React.createClass( {
```

Comment:
Split the components in bf17105 in two: `SiteTitleStep` which is the step itself and `SignupSiteTitle` which is the input field.